content/en/ecosystem/vendors.md

```diff
@@ -4,54 +4,39 @@ description: Vendors who natively support OpenTelemetry
-| Sentry                     | Yes               | No          | [sentry.io/...](https://sentry.io/for/opentelemetry/)
-| Sentry Software            | Yes               | Yes         | [sentrysoftware.com/...](https://www.sentrysoftware.com/products/hardware-sentry-opentelemetry-collector.html)
-| ServicePilot               | No                | Yes         | [servicepilot.com/...](https://www.servicepilot.com/en/doc/apm#opentelemetry)
-| SigNoz                     | Yes               | Yes         | [signoz.io](https://signoz.io)
-| SolarWinds                 | Yes               | Yes         | [documentation.solarwinds.com/...](https://documentation.solarwinds.com/en/success_center/observability/default.htm#cshid=third-otel-integration)
-| Splunk                     | Yes               | Yes         | [splunk.com/blog/...](https://www.splunk.com/en_us/blog/conf-splunklive/announcing-native-opentelemetry-support-in-splunk-apm.html)
-| Sumo Logic                 | Yes               | Yes         | [help.sumologic.com/](https://help.sumologic.com/docs/apm/traces/quickstart/)
-| TelemetryHub               | No                | Yes         | [telemetryhub.com](https://app.telemetryhub.com/docs)
-| Teletrace                  | Yes               | Yes         | [docs.teletrace.io](https://docs.teletrace.io/)
-| ThousandEyes (Cisco)       | No                | Yes         | [docs.thousandeyes.com](https://docs.thousandeyes.com/product-documentation/api/opentelemetry)
-| Traceloop                  | No                | Yes         | [traceloop.com](https://www.traceloop.com)
-| Uptrace                    | Yes               | Yes         | [uptrace.dev](https://uptrace.dev)
-
-\* _Vendors are listed alphabetically_.
+A non-exhaustive list of organizations, offering solutions that consume
+OpenTelemetry natively via via [OTLP](/docs/specs/otlp/), such as observability
+backends and observability pipelines.
+
+These solutions may be open source, commercial, or both. Some organizations
+provide a [distribution](/docs/concepts/distributions/) (of customized
+OpenTelemetry components), that provides additional capabilities or for improved
+ease of use.
+
+{{% ecosystem/vendor-table %}}
+
+To have your organization listed, submit a PR with an entry added to the
+[vendors list](https://github.com/open-telemetry/opentelemetry.io/tree/main/data/ecosystem/vendors.yaml).
+The entry must include the following[^grace-period-2024-01-01]:
+
+- a link to the documentation on how your offering consumes OpenTelemetry
+  natively via [OTLP](http://localhost:1313/docs/specs/otlp/).
+- if applicable, a link to your distribution.
+- if applicable, a link that proves that your offering is open source. An open
+  source distribution does not qualify your offering to be marked "open source".
+- a point of contact (GitHub handle or email address), we can reach out for
+  future inquiries.
```

Comment:
```suggestion
- Link to the documentation that details how your offering consumes OpenTelemetry
  natively via [OTLP](http://localhost:1313/docs/specs/otlp/).
- Link to your distribution, if applicable
- Link that proves that your offering is open source, if applicable. An open
  source distribution does not qualify your offering to be marked "open source".
- GitHub handle or email address as a point of contact so that we can reach out
  in case we have questions
```